xbmc/games/dialogs/osd/DialogGameVideoFilter.cpp

```diff
@@ -118,6 +118,9 @@ void CDialogGameVideoFilter::InitVideoFilters()
   {
     VideoFilterProperties videoFilter;
 
+    if (child->FirstChild() == nullptr)
+      continue;
```

Comment:
Can you also add nullptr checks to the FirstChild() calls below?

Reply:
All 8?

Reply:
Yea. There's an XMLUtils library that might cut down on some of the code.

Reply:
I didn't use it so that we can move the parsing code to the add-on at some point.

Reply:
OK, then we should add the nullptr checks to the code as it is

Reply:
Ok